dts/arm/infineon/psoc6/mpns/CY8C6244LQI_S4D92.dtsi

```diff
@@ -8,8 +8,6 @@
 #include <arm/armv7-m.dtsi>
 #include "../psoc6_04/psoc6_04.68-qfn.dtsi"
 
-/delete-node/ &scb3;
```

Comment:
CY8C6244LQI_S4D92 does not have scb3, that why we have  '/delete-node/ &scb3;'. What kind of problem do you have if do not remove it..?

Reply:
I was getting an error on `/delete-node/ &scb3;` because scb3 was not defined in the first place anywhere in the device tree files for this chip.

Reply:
ok. checked this. All, SOC_DIE_PSOC6_04 does not have scb3. so we do not have scb3 in dts\arm\infineon\psoc6\psoc6_04\psoc6_04.dtsi and we do not need `/delete-node/ &scb3;` in MPNs dtsi for SOC_DIE_PSOC6_04.

i will file the ticket to remove `/delete-node/ &scb3;` for all mpns of SOC_DIE_PSOC6_04